LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2023, Niels Mündler <n.muendler@posteo.de>
```

Comment:
@KtorZ Don't you want to add yourself here as well?

Reply:
No, I give it you 😅
I've got too many projects to maintain already.